src/types/index.ts

```diff
@@ -35,11 +35,13 @@ export type GetLocationsParams = {
 export type ConnectBluetoothReaderParams = {
   reader: Reader.Type;
   locationId?: string;
+  autoReconnect: boolean;
```

Comment:
looks like this param is named `autoReconnectOnUnexpectedDisconnect` on the native SDKs, can we stick to that here as well?